tests/lib/rules/no-trailing-spaces.js

```diff
@@ -416,6 +424,66 @@ ruleTester.run("no-trailing-spaces", rule, {
                     column: 8
                 }
             ]
+        },
+
+        // Tests for ignoreComments flag.
+        {
+            code: "var foo = 'bar'; ",
+            output: "var foo = 'bar';",
+            options: [{ ignoreComments: true }],
+            errors: [
+                {
+                    message: "Trailing spaces not allowed.",
+                    type: "Program",
+                    line: 1,
+                    column: 17
+                }
+            ]
+        },
+        {
+            code: "// Trailing comment test. ",
+            output: "// Trailing comment test.",
+            options: [{ ignoreComments: false }],
+            errors: [
+                {
+                    message: "Trailing spaces not allowed.",
+                    type: "Program",
+                    line: 1,
+                    column: 26
+                }
+            ]
+        },
+        {
+            code: "/* \nTrailing comments test. \n*/",
+            output: "/*\nTrailing comments test.\n*/",
+            options: [{ ignoreComments: false }],
+            errors: [
+                {
+                    message: "Trailing spaces not allowed.",
+                    type: "Program",
+                    line: 1,
+                    column: 3
+                },
+                {
+                    message: "Trailing spaces not allowed.",
+                    type: "Program",
+                    line: 2,
+                    column: 24
+                }
+            ]
+        },
+        {
+            code: "#!/usr/bin/env node ",
```

Comment:
Sorry, I know I asked for this, but I'm realizing now that what I actually meant was for shebangs to be totally ignored. What do you think?